BUILDING.md

```diff
@@ -1,7 +1,7 @@
-# Building NUnit 3.0
+# Building NUnit 3.0 Console and Engine
 
-NUnit 3.0 consists of three separate layers: the Framework, the Engine and the Console Runner. 
-The source code is kept in a single GitHub repository at http://github.com/nunit/nunit.git.
+NUnit 3.0 consists of two parts: the Engine and the Console Runner. 
+The source code is kept in a single GitHub repository at http://github.com/nunit/nunit-console.git.
```

Comment:
How about "NUnit 3.0 consists of three separate layers: the Framework, the Engine and the Console Runner. This repository contains the Engine and Console Runner."